dspace/config/xmlui.xconf

```diff
@@ -182,7 +182,7 @@
         <!-- <theme name="Test Theme 2" regex="community-list" path="theme2/"/> -->
 
         <!-- Mirage theme, @mire contributed theme, default since DSpace 3.0 -->
-        <theme name="Atmire Mirage Theme" regex=".*" path="Mirage/" />
+        <theme name="Atmire Mirage Theme" regex=".*" path="Mirage2/" />
```

Comment:
This minor issue still remains. We should not be changing the default theme. Please remove this modification to the xmlui.xconf